content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx

```diff
@@ -470,6 +471,44 @@ const usersWithCount = await prisma.user.findMany({
 </cmdResult>
 </CodeWithResult>
 
+#### Filter the relation count
+
+This feature is available from version 4.1.0. To use it, you need to add the [preview feature](/concepts/components/preview-features/client-preview-features#enabling-a-prisma-client-preview-feature) `filteredRelationCount`.
+
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["filteredRelationCount"]
+}
+```
+
+Use `where` to filter the fields returned by the `_count` output type. You can do this on [scalar fields](/concepts/components/prisma-schema/data-model#scalar-fields), [relation fields](/concepts/components/prisma-schema/data-model#relation-fields) and fields of a [composite type](/concepts/components/prisma-schema/data-model#defining-composite-types).
+
+For example:
```

Comment:
Could consider spelling it out? 'For example, the following query does bla bla bla...'

Reply:
Done!